.github/workflows/e2e-suite.yml

```diff
@@ -99,8 +103,14 @@ jobs:
           mv calicoctl-linux-amd64 /usr/local/bin/calicoctl
           mv kubectl /usr/local/bin/kubectl
 
+      - name: Set OpenAPI Spec URL
+        if: ${{ inputs.openapi_spec_url != '' }}
+        run: |
+          echo "Using OpenAPI Spec URL: ${{ inputs.openapi_spec_url }}"
+          export OPENAPI_SPEC_URL="${{ inputs.openapi_spec_url }}"
```

Comment:
Thank you for addressing my feedback! One more thing, I'm not sure if the environment is carried over between runs so you may need to echo this to `$GITHUB_ENV` like we do on line 119 & 134